exercises/etl/src/test/java/EtlTest.java

```diff
@@ -27,6 +27,7 @@ public void testTransformOneValue() {
         assertEquals(expected, etl.transform(old));
     }
 
+    @Ignore
```

Comment:
Thank you!
It's my first time making contribution on a real project. My apologies. I'll pay more attention next time.